tasks/publisher.yml

```diff
@@ -14,7 +14,7 @@
 - name: Copy certificate key on publisher 
   ansible.builtin.copy:
     content: "{{ item.crt }}"
-    dest: /var/tmp/keys/
+    dest: "{{ item.path }}"
```

Comment:
What if `item.path` base directory does not exist?

Which tasks use this certificate key?

Reply:
I mean, with your example, what happens if the directory `/path/to/certs/` does not exist?

Reply:
i understood i will add a check state of directory if exist will do the copy else will  not pass through .
```
- name: Get stats of the FS object
  ansible.builtin.stat:
    path: /path/to/something
  register: p
```

Reply:
Can't you just force `/var/tmp/keys/mykey`?

Reply:
i can do that but make sure when the directory does not exist will be an error Also.

Reply:
fixed